programmers/난이도별/level01.K번째수/dkdlelk99.py

```diff
@@ -0,0 +1,20 @@
+def solution(array, commands):
+    answer = []
+    for i in range(len(commands)):
```

Comment:
```suggestion
    for v in commands:
```
이렇게 commands의 요소로 반복문을 돌리고